app/Http/Controllers/V2/Tasks/ViewTaskUuidController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Controllers\V2\Tasks;
+
+use App\Http\Controllers\Controller;
+use App\Models\V2\Tasks\Task;
+
+class ViewTaskUuidController extends Controller
```

Comment:
I didn't even realize when I was reading over the FE PR that this is a whole new endpoint. Definitely should be replaced with work in the v3 repo in that case.